.github/workflows/ci.yml

```diff
@@ -167,4 +167,48 @@ jobs:
         run: /usr/bin/entrypoint tests
 
       - name: Build packages
-        run: /usr/bin/entrypoint package
\ No newline at end of file
+        run: /usr/bin/entrypoint package
+
+# Before merging the PR we must remove this, right now we keep it just to have packages also from there
+  build-modern-bpf-packages:
+    runs-on: ubuntu-22.04
+    steps:
+
+      - name: Install deps ⛓️
+        run: |
+          sudo apt update
+          sudo apt install -y --no-install-recommends ca-certificates cmake build-essential clang-14 git pkg-config autoconf automake libtool libelf-dev
+
+      - name: Checkout Falco 🦅
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+
+      # Build the modern BPF skeleton
+      - name: Build modern BPF skeleton 🐝
+        run: |
+          mkdir -p skeleton-build
+          cd skeleton-build && cmake -DUSE_BUNDLED_DEPS=ON -DBUILD_FALCO_MODERN_BPF=ON -DCREATE_TEST_TARGETS=Off ../
+          make ProbeSkeleton
```

Comment:
@Andreagit97: Would building on ubuntu-22.04 work for example for running Falco w/ modern_bpf on let's say a centos7 6.0 kernel? When we debugged the possible "heisenbug" I recall it would need to be built in a centos7 container because of old GLIBC version `2.17` constraints (given the modern probe is baked into scap). In that case need to build newer clang versions from source in the centos7 container or maybe curl pre-built clang artifacts from Falco's Artifact store?

Reply:
Or given below when you use the centos7 container this is all not a problem and it works. In that case please disregard.

Reply:
yeah, the trick here is to build the header file (so the BPF skeleton) on a recent machine with the latest clang/llvm versions like ubuntu 22.04 since building it directly on centos7 would be a real pain. After that, we use this header file to build Falco on centos7 in this way the modern probe should be compliant with lower versions of GLIBC like `2.17` as you mentioned :)